test/e2e-tests-upgrade.sh

```diff
@@ -85,6 +101,19 @@ go_test_e2e -timeout=20m ./test || failed=1
 # the upgrade.
 go_test_e2e -tags=examples -timeout=20m ./test/ || failed=1
 
+# Testing main against the latest release to detect regression.
+# TODO: #5782 add previous LTS and supported releases
+header "Testing main against the latest release to detect regression"
+get_tests_from_release $PREVIOUS_PIPELINE_VERSION
+
+skip_breaking_tests
```

Comment:
@JeromeJu curious if you've been able to test this locally, especially the test skipping part. Do you have an example output?

Reply:
maybe just comment in this thread with a link to a github gist?

Reply:
[failed test with `TestExamples/v1/pipelineruns/pipelinerun`, `TestWaitCustomTask_V1Beta1_PipelineRun`, which are known due to config not at local](https://gist.github.com/JeromeJu/4f0cab3ffbc5f8171434fe3cc1aa94e5)

[skip tests failed - after](https://gist.github.com/JeromeJu/bfdc08dea7dd12149a8d4a52605a38e0)